test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -48,6 +49,13 @@ TEST(QuicPlatformTest, QuicArraysize) {
 
 enum class TestEnum { ZERO = 0, ONE, TWO, COUNT };
 
+TEST(QuicPlatformTest, QuicBugTracker) {
+  EXPECT_DEBUG_DEATH(QUIC_BUG << "Here is a bug,", "");
+  EXPECT_DEBUG_DEATH(QUIC_BUG_IF(true) << "There is a bug,", "");
+  QUIC_PEER_BUG << "Everywhere's a bug,";
```

Comment:
expect log contains macro checks?

Also checks for if(false) not happening would be good.

Reply:
Done. PTAL.